### DISCUSSION TITLE
How to disable hydration completely?

### DISCUSSION THREAD

**Discussion by thejspro** [gatsbyjs/gatsby#29422] [Help]
I love the workflow and simplicity of working with Gatsby to create static websites, however for my use case once the website has been generated I do not need it to be rehydrated at all - my website is meant to be entirely static and it should not load React in the browser.

One could say that I wish to use Gatsby to render my website to a static HTML and leave it at that.

Is this possible at all?

**Comment by KyleAMathews** [comment]
This plugin is one way of doing this https://www.gatsbyjs.com/plugins/gatsby-plugin-no-javascript-utils/?=javasc

**Reply by thejspro** [reply]
Thank you, I will have a look.
